build/release.js

```diff
@@ -1,3 +1,4 @@
+var fs = require( "fs" );
```

Comment:
Feels very specific as "gotcha" style rule i.e. hard to follow, don't recall we do this in anywhere else.